corehq/apps/locations/templates/locations/location_types.html

```diff
@@ -108,7 +108,17 @@
                             </th>
                             <!-- /ko -->
                             {% if commtrack_enabled %}
-                            <th>{% trans "Tracks Stock" %}</th>
+                            <th>
+                                {% trans "Tracks Stock" %}
+                                <span class="hq-help-template"
+                                      data-title="{% trans "Tracks Stock" %}"
+                                      data-content="{% blocktrans %}
+                                          Does this location keep inventory and track stock?
+                                          Note that changing this property will update all
+                                          locations of this level accordingly.
```

Comment:
Sounds good.  @sheelio, I'm not sure how specific I should be here, but basically what I want to convey is that changing this property will affect all locations of that type (ie use with caution).